### PR TITLE
Distributed Kafka-ML

### PR DESCRIPTION
**Pull request by alexcarnero97**
Extension that allows working with distributed models.

### CODE REVIEW

File: model_training/distributed_training.py [hunk 1]
```diff
@@ -0,0 +1,308 @@
...
+                        for index in range(0, N-1):
+                            s = outputs[index]
+                            predictions.append(s[1])
+                        predictions.append(outputs[-1])
+                        """Obteins all the prediction outputs from each distributed submodel"""
+
+                        model = keras.Model(inputs=[img_input], outputs=predictions, name='model')
+                        """Creates a global model consisting of all distributed submodels"""
+
+                        weights = {}
```
Comment (DRTorresRuiz) [inline model_training/distributed_training.py:190]: This variable is called `weights`, but stores the loss function name for each model.
